nucypher/utilities/clouddeploy.py

```diff
@@ -0,0 +1,715 @@
+                    return False
+
+            if self.config.get('InternetGateway'):
+                self.ec2Resource.InternetGateway(self.config['InternetGateway']).detach_from_vpc(VpcId=self.config['Vpc'])
+                self.ec2Resource.InternetGateway(self.config['InternetGateway']).delete()
+                self.emitter.echo(f'deleted InternetGateway: {self.config["InternetGateway"]}')
+                del self.config['InternetGateway']
+            self._write_config()
+
+            if self.config.get('Vpc'):
+                vpc.delete()
+                self.emitter.echo(f'deleted Vpc: {self.config["Vpc"]}')
+                del self.config['Vpc']
+
+            if self.config.get('keypair'):
+                self.emitter.echo(f'deleting keypair {self.keypair} in 5 seconds...', color='red')
+                time.sleep(6)
+                self.ec2Client.delete_key_pair(KeyName=self.config.get('keypair'))
+                del self.config['keypair']
+                os.remove(self.config['keypair_path'])
+
+        return True
+
+    def create_new_node_for_staker(self, address):
+        new_instance_data = self.ec2Client.run_instances(
+            ImageId=self.EC2_AMI,
+            InstanceType=self.EC2_INSTANCE_SIZE,
+            MaxCount=1,
+            MinCount=1,
+            KeyName=self.keypair,
+            NetworkInterfaces=[
+                {
+                    'AssociatePublicIpAddress': True,
+                    'DeleteOnTermination': True,
+                    'DeviceIndex': 0,
+                    'Groups': [
+                        self.config['SecurityGroup']
+                    ],
+                    'SubnetId': self.config['Subnet'],
+                },
+            ],
+            TagSpecifications=[
+                {
+                    'ResourceType': 'instance',
+                    'Tags': [
+                        {
+                            'Key': 'Name',
+                            'Value': f'{self.namespace}-{address}'
+                        },
+                    ]
+                },
+            ],
+        )
+
+        self.config['instances'][address] = {
+            'InstanceId': new_instance_data['Instances'][0]['InstanceId'],
+        }
+
+        instance = self.ec2Resource.Instance(new_instance_data['Instances'][0]['InstanceId'])
+        self.emitter.echo("\twaiting for instance to come online...")
+        instance.wait_until_running()
+        instance.load()
+        self.config['instances'][address]['publicaddress'] = instance.public_dns_name
+        if self.config['seed_network'] and not self.config.get('seed_node'):
+            self.config['seed_node'] = instance.public_dns_name
+        self._write_config()
+
+
+class CloudDeployers:
+
+    aws = AWSNodeConfigurator
+    do = DigitalOceanConfigurator
+    digitalocean = DigitalOceanConfigurator
```

Comment:
Any reason we have 2 for digital ocean? Is it to allow `do` or `digitalocean`? Perhaps whould we just choose one - `digitalocean` is probably more descriptive.

Reply:
Can we get rid of the `do` configurator?